version_with_class.py

```diff
@@ -0,0 +1,191 @@
+import os
+import tempfile
+from functools import wraps
+import random
+import subprocess
+
+from typing import Dict, Tuple, List, Generator, Callable, List
+from iib.exceptions import IIBError, AddressAlreadyInUse
+
+
+class Config:
+    iib_opm_port_ranges: Dict[str, Tuple[int, int]] = {
+        "opm_port": (50051, 50151),
+        "opm_ppof_port": (50151, 50251),
+    }
+    iib_opm_pprof_lock_required_versions = ["1.40.0"]
```

Comment:
Let's not keep a list of versions with pprof. I believe that it will be present there from now on. And it looks like it is there since opm version [v1.29.0](https://github.com/operator-framework/operator-registry/releases/tag/v1.29.0)